tasks/Resources/Provisioning/Hive/AWS/oc-provision.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Library           OperatingSystem
+
+*** Keywords ***
+Run AWS Configuration In Hive
+    [Arguments]    &{infrastructure_configurations}
+    Oc Apply    kind=List    src=tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml
+    ...    template_data=${infrastructure_configurations}
+    Sleep    60
+    ${namespace} =    Oc Get    kind=Namespace    label_selector=hive.openshift.io/cluster-pool-name=${infrastructure_configurations}[hive_cluster_name]
+    Log    ${namespace[0]['metadata']['name']}    console=True
+    ${pod} =    Oc Get    kind=Pod    namespace=${namespace[0]['metadata']['name']}
+    Log    ${pod[0]['metadata']['name']}    console=True
+    Wait Until Keyword Succeeds    40 min    10 s
+    ...    Wait Until Cluster Is Provisioned    ${pod[0]['metadata']['name']}    ${namespace[0]['metadata']['name']}
+
+Wait Until Cluster Is Provisioned
```

Comment:
`Wait Until Cluster Is Provisioned` is not really waiting. I think you should do something like this:
```
Cluster Should Be Provisioned
    [Arguments]    ${pod_name}    ${namespace}
    ${installation_log} =    Oc Get Pod Logs    name=${pod_name}    container=hive    namespace=${namespace}
    Should Contain    ${installation_log}    install completed successfully

Wait Until Cluster Is Provisioned
    [Arguments]    ${pod_name}    ${namespace}
    Wait Until Keyword Succeeds    40 min    10 s
    ...    Cluster Should Be Provisioned    ${pod_name}    ${namespace}

```